### PR TITLE
feat(builtin.copy): add simple ignore option

### PR DESCRIPTION
**Pull request by JSerFeng**
## Summary

Add simple ignore option for copy-plugin.

## Types of changes

- [ ] Docs change / Dependency upgrade
- [ ] Bug fix
- [x] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [x] I have added changeset via `pnpm run changeset`.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: crates/rspack_core/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ dashmap = "5.0.0"
 derivative = "2.2.0"
 dyn-clone = "1.0.10"
 futures = "0.3.21"
+glob = "0.3.1"
 glob-match = "0.2.1"
```
Comment (IWANABETHATGUY) [inline crates/rspack_core/Cargo.toml:19]: Would you mind explaining why introduce `glob` since we already have `glob-match` ?

Comment (JSerFeng) [reply crates/rspack_core/Cargo.toml:19]: glob-match has no custom configuration, although glob's configuration is not that much

**Review by IWANABETHATGUY — APPROVED** [review]

**Comment by hardfist** [comment]
please fix conflict

**Comment by JSerFeng** [comment]
> please fix conflict

Done

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**